src/main/kotlin/no/nav/syfo/maskinporten/MaskinportenTokenApi.kt

```diff
@@ -28,45 +29,46 @@ fun Routing.registerMaskinportenTokenApi(
     route("/api/test/token") {
         authenticate("test-token") {
             get {
+                val log = call.application.log
                 val jwtGrant = generateJwtGrant(env)
                 val maskinportenTokenUrl = env.auth.maskinporten.tokenUrl
+
                 val response: HttpResponse = httpClient.submitForm(
                     url = maskinportenTokenUrl,
                     formParameters = parameters {
                         append("grant_type", "urn:ietf:params:oauth:grant-type:jwt-bearer")
                         append("assertion", jwtGrant)
                     }
                 )
+
                 val maskinportenResponse: MaskinportenResponse = response.body()
-                val maskinPortenAccessToken = maskinportenResponse.access_token
-                maskinPortenAccessToken?.let { token ->
+
+                if (maskinportenResponse.access_token != null) {
+                    call.respond(HttpStatusCode.OK, maskinportenResponse.access_token)
+                } else {
+                    log.error(
+                        "Token error: ${maskinportenResponse.error} " +
```

Comment:
`maskinportenResponse.error` og `maskinportenResponse.error_description` kan kanskje bli variabler siden de er brut to ganger

Reply:
Fikser!